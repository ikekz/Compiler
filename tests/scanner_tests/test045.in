absolute
and 
array 
asm 
begin 
case 
const 
constructor 
destructor 
dispose 
div 
do 
downto
else 
end 
exit 
false 
file 
for 
function 
goto 
if 
implementation 
in 
inherited 
inline
interface 
label 
mod 
new 
nil 
not 
object 
of 
operator 
or 
packed 
procedure 
program
record 
reintroduce 
repeat 
self 
set 
shl 
shr 
string 
then 
to 
true 
type 
unit 
until
uses 
var 
while 
with 
xor