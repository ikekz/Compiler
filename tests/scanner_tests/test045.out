1   1   reserved word  absolute       absolute
2   1   reserved word  and            and
3   1   reserved word  array          array
4   1   reserved word  asm            asm
5   1   reserved word  begin          begin
6   1   reserved word  case           case
7   1   reserved word  const          const
8   1   reserved word  constructor    constructor
9   1   reserved word  destructor     destructor
10  1   reserved word  dispose        dispose
11  1   reserved word  div            div
12  1   reserved word  do             do
13  1   reserved word  downto         downto
14  1   reserved word  else           else
15  1   reserved word  end            end
16  1   reserved word  exit           exit
17  1   reserved word  false          false
18  1   reserved word  file           file
19  1   reserved word  for            for
20  1   reserved word  function       function
21  1   reserved word  goto           goto
22  1   reserved word  if             if
23  1   reserved word  implementation implementation
24  1   reserved word  in             in
25  1   reserved word  inherited      inherited
26  1   reserved word  inline         inline
27  1   reserved word  interface      interface
28  1   reserved word  label          label
29  1   reserved word  mod            mod
30  1   reserved word  new            new
31  1   reserved word  nil            nil
32  1   reserved word  not            not
33  1   reserved word  object         object
34  1   reserved word  of             of
35  1   reserved word  operator       operator
36  1   reserved word  or             or
37  1   reserved word  packed         packed
38  1   reserved word  procedure      procedure
39  1   reserved word  program        program
40  1   reserved word  record         record
41  1   reserved word  reintroduce    reintroduce
42  1   reserved word  repeat         repeat
43  1   reserved word  self           self
44  1   reserved word  set            set
45  1   reserved word  shl            shl
46  1   reserved word  shr            shr
47  1   reserved word  string         string
48  1   reserved word  then           then
49  1   reserved word  to             to
50  1   reserved word  true           true
51  1   reserved word  type           type
52  1   reserved word  unit           unit
53  1   reserved word  until          until
54  1   reserved word  uses           uses
55  1   reserved word  var            var
56  1   reserved word  while          while
57  1   reserved word  with           with
58  1   reserved word  xor            xor
